packages/sindarian-server/src/guards/decorators/use-guards-decorator.test.ts

```
@@ -99,9 +99,7 @@ describe('GuardHandler.execute', () => {
   it('should handle async guards', async () => {
     class AsyncGuard implements CanActivate {
       async canActivate(): Promise<boolean> {
-        return new Promise((resolve) =>
-          setTimeout(() => resolve(true), 10)
-        )
+        return new Promise((resolve) => setTimeout(() => resolve(true), 10))
```

ESLint

[error] Pass function and arguments directly to timer function to avoid allocating an extra function

(e18e/prefer-timer-args)